vault_program/src/initialize_vault.rs

```diff
@@ -36,11 +38,23 @@ pub fn process_initialize_vault(
     load_system_account(vault, true)?;
     load_system_account(vrt_mint, true)?;
     load_signer(vrt_mint, true)?;
-    load_token_mint(mint)?;
+    load_token_mint(st_mint)?;
     load_signer(admin, true)?;
     load_signer(base, false)?;
     load_system_program(system_program)?;
     load_token_program(token_program)?;
+    load_token_account(
```

Comment:
nit: put these inline with arguments above

Reply:
do in order of accounts passed in

Reply:
Done